examples/fungible_asset/main.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aptos-labs/aptos-go-sdk"
+	"github.com/aptos-labs/aptos-go-sdk/bcs"
+	"github.com/aptos-labs/aptos-go-sdk/crypto"
+	"github.com/aptos-labs/aptos-go-sdk/internal/util"
+)
+
+const testEd25519PrivateKey = "0xc5338cd251c22daa8c9c9cc94f498cc8a5c7e1d2e75287a5dda91096fe64efa5"
+const testEd25519PublicKey = "0xde19e5d1880cac87d57484ce9ed2e84cf0f9599f12e7cc3a52e4e7657a763f2c"
+const testEd25519Address = "0x978c213990c4833df71548df7ce49d54c759d6b6d932de22b24d56060b7af2aa"
+
+// These come from fungible_asset.json
```

Comment:
embed the json into the binary so that it can be read and parsed?
https://pkg.go.dev/embed

Reply:
This ideally would be built on a case by case basis, embedding in the binary could be interesting, but also less realistic.